treeherder/model/management/commands/cycle_data.py

```diff
@@ -101,6 +101,11 @@ def __init__(
         self.max_runtime = max_runtime or PerfherderCycler.DEFAULT_MAX_RUNTIME
         self.strategies = strategies or RemovalStrategy.fabricate_all_strategies(chunk_size)
 
+    @property
+    def max_timestamp(self):
```

Comment:
`max_timestamp` is ambiguous. We need to rename this property to something more specific.